recipes/tutorials/export_a_kili_project.ipynb

```diff
@@ -0,0 +1,888 @@
+{
```

Comment:
<p>The output shows an error: <code>Project needs at least one OBJECT_DETECTION task.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/885/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
I just ran it twice, once in jupyter lab, once in pytest, it ran without any issues.

did you run the notebook on master?